projects/js-packages/components/components/record-meter-donut/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import './style.scss';
+export type RecordMeterDonutProps = {
+	/**
+	 * Total number of items for the record meter donut.
+	 */
+	totalCount: number;
+	/**
+	 * Count for the given item
+	 */
+	segmentCount: number;
+	/**
+	 * Color code for the background color for the item.
+	 * If not provided, defaults to Jetpack Green
+	 */
+	backgroundColor?: string;
+
+	/**
+	 * thickness for the chart border
+	 * If not provided, defaults to 3.5
+	 */
+	thickness?: string;
+	/**
+	 * width for the full chart size
+	 * If not provided, defaults to 64px
+	 */
+	donutWidth?: string;
+};
+
+/**
+ * Generate record meter donut bar
+ *
+ * @param {RecordMeterDonutProps} props - Props
+ * @returns {React.ReactElement} - JSX element
+ */
+const RecordMeterDonut: React.FC< RecordMeterDonutProps > = ( {
+	totalCount,
+	segmentCount,
+	backgroundColor = '#00BA37', // jetpack green fallback
+	thickness = '3.5',
+	donutWidth = '64px',
+} ) => {
+	const count = () => {
+		// get count as a percent value
+		return ( segmentCount / totalCount ) * 100;
+	};
+
+	return (
+		<div className="record-meter-donut" aria-hidden="true">
+			<svg
+				width={ donutWidth }
+				height="auto"
+				viewBox="0 0 40 40"
+				className="record-meter-donut_svg"
+				data-testid="record-meter-donut_svg"
+			>
+				<circle
+					className="record-meter-donut-hole"
+					cx="20"
+					cy="20"
+					r="15.91549430918954"
+					fill="transparent"
+				></circle>
+				<circle
+					className="record-meter-donut-ring"
+					cx="20"
+					cy="20"
+					r="15.91549430918954"
+					fill="transparent"
+					strokeWidth={ thickness }
+					stroke="#ebebeb"
+				></circle>
+				<circle
+					className="record-meter-donut-segment"
+					cx="20"
+					cy="20"
+					r="15.91549430918954"
```

Comment:
I'm curious - where does this number come from?

Reply:
I've added some comments into the code to help :)